errata_tool/erratum.py

```diff
@@ -211,6 +212,7 @@ def _fetch(self, errata_id):
             # NEW_FILES QE etc.
             self.errata_state = erratum['status']
             self._original_state = self.errata_state
+            self._original_json = erratum
```

Comment:
This is only capturing the API/v1 content. The old style errata content is not here. It's entirely fine if that's what you need, but I just wanted to make a note of that.

Reply:
There are a few fields that are not in the API/v1 content; see line 270-ish.